build/validate-policies.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+KUBECONFORM=kubeconform
+KC_VERSION=v0.5.0
+KUSTOMIZE_VERSION=v4.5.7
+GENERATOR_PATH=policy.open-cluster-management.io/v1/policygenerator
+
+export GOBIN=${PWD}/bin
+export PATH=${GOBIN}:${PATH}
+mkdir -p ${GOBIN}
+
+OUT=`which kubeconform`
+RC=$?
+
+set -euo pipefail  # exit on errors and unset vars, and stop on the first error in a "pipeline"
+
+# Validate the policy resource file under the directory provided
+validatePolicies() {
+	find $1 -name "*.yaml" | xargs kubeconform -schema-location 'schemas/{{ .ResourceKind }}_{{ .ResourceAPIVersion }}.json' -summary
+}
+
+# Validate the generator based policyset projects under the directory provided
+validatePolicySets() {
+	KPATH=`pwd`
+	SETPATH=policygenerator/policy-sets/$1
+	cd $SETPATH
+	for project in `find . -name "kustomization.y*ml" | sed 's,/kustomization.*,,g'`; do
+	        cd $project
+		echo "Generating PolicySet $project"
+	        OUTPUT=`kustomize build --enable-alpha-plugins`
+	        cd $KPATH/$SETPATH
+	done
+	cd $KPATH
+}
+
+if [ $RC -ne 0 ]; then
+	go install github.com/yannh/kubeconform/cmd/kubeconform@$KC_VERSION
+fi
+
+if [ ! -d schemas ]; then
+	echo "Getting Schema conversion tool"
+	mkdir schemas
+	cd schemas
+	curl -o crd-schema.py https://raw.githubusercontent.com/yannh/$KUBECONFORM/$KC_VERSION/scripts/openapi2jsonschema.py
+	chmod a+x crd-schema.py
+
+	echo "Converting CRDs to Schemas"
+	./crd-schema.py https://raw.githubusercontent.com/open-cluster-management-io/governance-policy-propagator/main/deploy/crds/policy.open-cluster-management.io_policies.yaml
+	./crd-schema.py https://raw.githubusercontent.com/open-cluster-management-io/governance-policy-propagator/main/deploy/crds/policy.open-cluster-management.io_placementbindings.yaml
+	./crd-schema.py https://raw.githubusercontent.com/open-cluster-management-io/api/main/cluster/v1beta1/0000_02_clusters.open-cluster-management.io_placements.crd.yaml
```

Comment:
These should probably point to `stolostron` also. Maybe put it in a variable here and then update the workflow with `export GITHUB_ORG=${{ github.repository_owner }}`? That way, all files can be the same in all orgs.
```suggestion
	GITHUB_ORG=${GITHUB_ORG:-"open-cluster-management-io"}
	./crd-schema.py https://raw.githubusercontent.com/${GITHUB_ORG}/governance-policy-propagator/main/deploy/crds/policy.open-cluster-management.io_policies.yaml
	./crd-schema.py https://raw.githubusercontent.com/${GITHUB_ORG}/governance-policy-propagator/main/deploy/crds/policy.open-cluster-management.io_placementbindings.yaml
	./crd-schema.py https://raw.githubusercontent.com/${GITHUB_ORG}/api/main/cluster/v1beta1/0000_02_clusters.open-cluster-management.io_placements.crd.yaml
```

Reply:
I made an update to use a common org.  Hopefully it will match the right one if I understand the doc correctly. THanks!